device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256S1/buffers_defaults_objects.j2

```diff
@@ -148,6 +148,7 @@
 
 {%- macro generate_queue_buffers(port_names_active, port_names_inactive) %}
     "BUFFER_QUEUE": {
+{% if dynamic_mode is not defined %}
```

Comment:
please check if "endif" is needed here

Reply:
yes, it closes in line 205
pay attention that we have two `if`s wthe `endif` in the same line